ampersand-form-view.js

```diff
@@ -183,7 +187,7 @@ module.exports = View.extend({
             delete this._startingValues;
         }
         this.handleSubmit = this.handleSubmit.bind(this);
-        this.el.addEventListener('submit', this.handleSubmit, false);
+        this.formEl.addEventListener('submit', this.handleSubmit, false);
```

Comment:
Well, in my personal case it is: I want to wrap this view in a `<div>` like I do all my other views, rather than having to have a parent view just to wrap it in a `<div>`.

But rather than pushing for my personal use-case, I'd suggest that it's more so about allowing the view's `template` to be customized like other ampersand views (including InputView). By listening to `submit` events on `el`, you'd be creating a requirement that the root element of a FormView be a `<form>`, a philosophy that seems to run counter to [what was envisioned for the form view](https://ampersandjs.com/learn/forms/#so-what-do-we-do-in-ampersand-for-this):

> the nice thing about that is now we've decoupled `<input>`s from values. So if you wanted a password from the user, you would have a single "password-view" that actually rendered two <input>s, for password and validation of that password, but ultimately just produced a single value.
> 
> So rather than thinking of a "field view" as a single form control, we think of it as a view that's responsible for a certain form value, whatever widget that might be.